### ISSUE TITLE
[BUG] OpenSearch 2.3 (AWS) - When creating monitors the index selector does not show data stream indices

### ISSUE THREAD

**Issue by bzgj-nickel** [opensearch-project/alerting#782]
As in title

**Comment by lezzago** [comment]
There is no official support for data streams in the Alerting plugin. This will be a feature request to make sure there is proper full support here. 

Are you able to use the API to create the Alerting monitor with data streams?

**Comment by bzgj-nickel** [comment]
Yes, I am able to do it with API. In fact I have been doing this in 1.x.x version before recently upgrading OpenSearch.
More: in the 1.x.x it was working in UI as well :)
